### DISCUSSION TITLE
How can I write the optimization formula？

### DISCUSSION THREAD

**Discussion by paperballin** [typst/typst#1217] [Q&A] [answered]
I want to write a objective function and subjective function in typst, but I can't find a function to write this optimizatoin formula.

**Comment by xkevio** [comment]
Since not everyone is familiar with these terms, or what you want to write, can you provide an example? Perhaps, how it would look as an image or with LaTeX?

**Reply by paperballin** [reply]
![IMG_20230516_222931](https://github.com/typst/typst/assets/19199639/53fb9201-83cb-48e7-8e0c-20b124686690)
Like this..., is there an exsisting function to write this or I have to customize it？

**Reply by xkevio** [reply, marked as answer]
The following will produce this (only some minor differences regarding the `cal` font):
```
$ 
bold(P_1) quad min_(chi, cal(F), cal(P))&cal(Z)_(l x)(t) \
"s.t:" quad C 1:& alpha^m_(l x)(t), s^m_x(t), psi_u(t) in {0, 1}, forall l in cal(L), x in cal(X). \
C 2:& sum_(x in cal(X)) alpha_(l x)(t) <= 1, forall l in cal(L). \
C 3:& sum_(x in cal(X)) s^m_x(t) = 1, forall l in cal(L). \
C 4:& sum_(x in cal(X)) f_(l x)(t) <= F^(m a x)_x, forall l in cal(L). \
C 5:& sum_(x in cal(X)) p_(l x)(t) <= P^(m a x)_x, forall l in cal(L). \
C 6:& Upsilon^u(t) in [0, 1]
$
```

![image](https://github.com/typst/typst/assets/13004777/c879f637-dbc2-4c7e-bf48-4f074f2de6ce)

**Reply by paperballin** [reply]
Thank you! Now I know how to write it.
